src/bin/pgcopydb/cli_list.c

```diff
@@ -275,7 +277,7 @@ cli_list_db_getopts(int argc, char **argv)
 		exit(EXIT_CODE_BAD_ARGS);
 	}
 
-	while ((c = getopt_long(argc, argv, "S:D:s:t:F:xPL:fcCyarJRIN:Vdzvqh",
+	while ((c = getopt_long(argc, argv, "S:D:s:t:F:xPL:fcCyarJRIN:Vdzvqhk",
```

Comment:
Please put the option short letter at the right place in the string (same place as in the long options array).